nse_nsock.cc

```diff
@@ -1104,7 +1104,8 @@ LUALIB_API int luaopen_nsock (lua_State *L)
   int top = lua_gettop(L);
 
   /* library upvalues */
-  nsock_pool nsp = new_pool(L); /* NSOCK_POOL */
```

Comment:
Actually this is not a problem. I've just split the variable declaration and assignation to fix the following compiler warning: "Unused variable nsp". Thus, this is more to bypass the warning than to fix anything at all.